source/puppet/4.2/reference/upgrade_major_server.markdown

```diff
@@ -173,10 +173,10 @@ At this point, Puppet Server is ready to serve nodes in production. If you pulle
 
 In the pre-upgrade steps, you upgraded to PuppetDB 2.3.x, including the [`puppetdb-terminus`][puppetdb-terminus] package on your Puppet Servers. This means the upgraded Puppet Server can already communicate with your existing PuppetDB server.
 
-Now that you've upgraded Puppet Server, you can [upgrade PuppetDB][] to version 3 if you want. Be careful: it adds some cool improvements, but it also retires older API versions, which can break older integrations. See the [PuppetDB 3.0 release notes][] for details.
+Now that you've upgraded Puppet Server, you can [upgrade PuppetDB][] to version 3 if you want. Be careful: it adds some cool improvements, but it also retires older API versions, which can break older integrations. See the [PuppetDB 3.1 release notes][] for details.
```

Comment:
Is this actually what you want to do? The point of this link was to permanently link to the release notes that describe the breaking API changes from 3.0.0. If you're going to point to newer release notes, the link should include an anchor to make sure the reader can easily reach the proper notes.